appveyor.yml

```diff
@@ -1,22 +1,48 @@
-version: "{build}"
-build: off
-skip_tags: true
+build: false
+skip_branch_with_pr: true
 
 environment:
   matrix:
     - nodejs_version: "6"
     - nodejs_version: "8"
     - nodejs_version: "10"
+    - nodejs_version: "11"
+    - nodejs_version: "Current"
```

Comment:
This will slow the build down somewhat, as 11 === current

Reply:
Lets remove 11 then?